### PR TITLE
Implement lowering of aten.cosh op.

### PR DESCRIPTION
**Pull request by godot73**

### CODE REVIEW

File: test/Conversion/TorchToLinalg/elementwise.mlir
Comment (ramiro050) [inline test/Conversion/TorchToLinalg/elementwise.mlir:1]: No need to have this test. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

**Review by ramiro050 — APPROVED** [review]
Looks good, just one small change

**Comment by ramiro050** [comment]
The CI is failing because `AbstractInterpLib.cpp` does not match the autogenerated one. Run `./build_tools/update_abstract_interp_lib.sh` to regenerate the file.

**Comment by godot73** [comment]
Another CI test was failing around the GeneratedTorchOps.td. Resolved it by editing projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/torch_ods_gen.py instead of the .td file.

**Pull request merged**